tests/test_ui/test_ui_drag_and_drop.py

```diff
@@ -1,6 +1,5 @@
 import pytest
 
-
```

Comment:
You made no changes at all. Rewrite this test to make it less flaky and simpler.

Reply:
Thank you for the feedback. I will simplify the test and ensure it is less flaky by focusing on the core functionality and removing unnecessary complexity. I'll update the PR shortly.